pkg/capture/capture_test.go

```diff
@@ -58,48 +57,14 @@ func (t testMockSrcs) Wait() error {
 func TestConcurrentMethodAccess(t *testing.T) {
 	for _, i := range []int{1, 2, 3, 10} {
 		t.Run(fmt.Sprintf("%d ifaces", i), func(t *testing.T) {
-			testConcurrentMethodAccess(t, i, 100)
+			testConcurrentMethodAccess(t, i, 1000)
```

Comment:
One nice side effect: Due to all the optimizations I was able to bump this back up to 1000 iterations again (which I take as a good sign)!